### ISSUE TITLE
Use Lighthouse tables from BQ

### ISSUE THREAD

**Issue by ebidel** [ebidel/lighthouse-httparchive#3]
LH specific tables now: 
https://bigquery.cloud.google.com/table/httparchive:har.2017_06_20_android_lighthouse?pli=1&tab=preview

**Comment by ebidel** [comment]
Keeping this open b/c it isn't pushed yet. The 6/20 table only has 18 lh report, which is not enough to show meaningful data.

**Comment by rviscomi** [comment]
LH data in HTTP Archive has been collected regularly since 6/1 under the `_android_lighthouse` suffix. There's also a convenience table [latest_lighthouse_mobile](https://bigquery.cloud.google.com/table/httparchive:har.latest_lighthouse_mobile?tab=schema) which aliases the most recent results.

I've also generated a one-off table with a more LH specific schema, so you could query by category, audit, and score: [scratchspace.2017_06_15_lighthouse_audits](https://bigquery.cloud.google.com/table/httparchive:scratchspace.2017_06_15_lighthouse_audits?tab=preview). If this kind of table would be useful, let's chat about a schema that works best for you and I can help get the table generated regularly. Then on this end, we'd just need to modify the query to pull from the new tables.

**Comment by ebidel** [comment]
@brendankenny has been generating LH-specific tables too. Do you think we should has @rviscomi take that over for us?

**Comment by brendankenny** [comment]
yeah, I can share the query I use (it's big) if we want a more official table with a LH-specific schema, though I'll likely keep generating my side tables since we keep adding checks for things of LH-development interest (e.g. rates of specific error types) that change every time we release a new version
